doc/source/gettingstarted.rst

```diff
@@ -80,6 +80,7 @@ Step 3A: Installation for Kubernetes
    Uninstall::
 
     helm uninstall my-araali-agent
+    kubectl delete daemonset,namespace,serviceaccount,clusterrole,clusterrolebinding,deployment,service,secret,configmap,crd -l is_agent=true -A
```

Comment:
add a comment - the above only uinstalls the installer (due to a bug). Use the command below to uinstall all araali components